css/main.css

```diff
@@ -307,21 +307,153 @@ li {
 }
 
 .list-title {
-  font-family: "Inter", sans-serif;
+  font-family: 'Inter', sans-serif;
   font-style: normal;
   font-weight: bold;
   font-size: 32px;
   line-height: 44px;
 }
 
 .languages {
-  list-style: inside url("../media/svg/rectangle.svg");
+  list-style: inside url('../media/svg/rectangle.svg');
 }
 
 .frameworks {
-  list-style: inside url("../media/svg/square.svg");
+  list-style: inside url('../media/svg/square.svg');
 }
 
 .skills {
-  list-style: inside url("../media/svg/circle.svg");
+  list-style: inside url('../media/svg/circle.svg');
+}
+
+/* button styles */
+.btn {
+  font-weight: bold;
+  font-size: 17px;
+  line-height: 24px;
+  color: #f55800;
+  background: #fff;
+  border: 1px solid #7f8cff;
+  padding: 8px 10px;
+  width: fit-content;
+  transition: all 0.3s ease-in-out 0s;
+  display: flex;
+  align-items: center;
+}
+
+.btn:hover {
+  background: #f55800;
+  color: #fff;
+  box-shadow: 0 8px 16px rgba(64, 83, 252, 0.24);
+  cursor: pointer;
+}
+
+.btn:disabled {
+  color: #5e6c84;
+  border: 1px solid #5e6c84;
+}
+
+/* Contact Form */
+.footer {
+  background-color: #fff;
+  border-top-left-radius: 90px;
+}
+
+.footer .contact {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  padding: 100px 0;
+}
+
+.footer .contact h2 {
+  margin-top: 0;
+}
+
+.footer .contact h2,
+.footer .contact p {
+  text-align: initial;
+  color: #ebebff;
+}
+
+.footer .contact p {
+  font-size: 40px;
+  line-height: 52px;
+  transition: transform 2s ease-in-out 0s;
+  color: #262626;
+  font-family: Inter, Helvetica, sans-serif;
+  font-weight: bold;
+}
+
+.footer .contact p:hover {
+  transform: translate(20px);
+}
+
+.footer .contact form {
+  max-width: 400px;
+  margin: 20px auto;
+}
+
+.footer .contact form input,
+.footer .contact form textarea {
+  background: #fff;
+  padding: 10px 16px;
+  border: 1px solid #cfd8dc;
+  color: #172b4d;
+  font-size: 17px;
+  width: 100%;
+  margin: 10px 0;
+}
+
+.footer .contact form textarea {
+  max-width: 100%;
+  height: 170px;
+}
+
+.footer .contact form .btn {
+  margin-top: 10px;
+}
+
+.footer .contact form .error-msg {
+  color: yellow;
+}
+
+@media (min-width: 960px) {
+  .footer {
+    background-color: #fff;
+    text-align: center;
+  }
+
+  .footer h2,
+  .footer p {
+    max-width: 500px;
+    margin: auto;
+    text-align: center;
+  }
+
+  .footer form .btn {
+    margin: auto;
+  }
```

Comment:
- [ ] Required: Kindly make sure to centre the button at the bottom of the form as per Figma's design, colours, fonts and pattern. Pls, fix this issue.
Your design- 
<img width="501" alt="Screenshot 2022-03-09 at 11 42 41 PM" src="https://user-images.githubusercontent.com/70269507/157537586-6ad002b0-5a4e-4860-b7f6-9ae9dec45a3c.png">
Figma design-
<img width="267" alt="Screenshot 2022-03-10 at 12 12 00 AM" src="https://user-images.githubusercontent.com/70269507/157537651-e843ab45-1a22-4c65-959c-2f3d12f153ca.png">